### ISSUE TITLE
File cannot be extracted on Windows

### ISSUE THREAD

**Issue by tmontney** [ChrisYounger/config_explorer#20]
> File was uploaded but could not be extracted because the required application (tar/unzip) not found.

I've gotten this on Linux, where I installed unzip to fix the issue. On Windows, unzipping should be natively available, so I'm not sure why it can't extract.

**Comment by ChrisYounger** [comment]
Yeah unfortunatly this feature requires you to use the Linux subsystem in Windows.  You can enable it like so: 
 https://docs.microsoft.com/en-au/windows/wsl/install-win10#enable-the-windows-subsystem-for-linux-feature-gui

You can also install cygwin.

**Comment by tmontney** [comment]
It looks like you're calling it through line 47, subprocess. Why not just use PowerShell's _Expand-Archive_ or CMD and _CreateObject("Scripting.FileSystemObject")_?

**Comment by ChrisYounger** [comment]
yeah that seems like a good idea. I don't have any experience with Powershell, but if you did want to send me the code to make this work that I can include it.
